doc/api/api_changes/2016-10-17-AL_deprecations.rst

```diff
@@ -0,0 +1,11 @@
+Removal of deprecated features
+------------------------------
+
+The ``matplotlib.backends.qt_compat`` module has been removed.
+
+The ``set_graylevel`` method has been removed from the ``GraphicsContextBase``
+class and its subclasses.  The ``toggle_axisline`` method has been removed from
+the ``mpl_toolkits.axes_grid1.mpl_axes.Axes`` class.
+
+Support for setting an ``Axes``' aspect to ``normal"`` has been removed, in
```

Comment:
wow… OK.
Yeah, I think that's proper english, but you know… french people and english…